From 872d05e296709d1133bd8dbc26ae4cf31f03ac89 Mon Sep 17 00:00:00 2001
From: Stefan Hansson <newbyte@postmarketos.org>
Date: Mon, 1 Apr 2024 18:13:59 +0200
Subject: [PATCH thumbdrives 1/2] Migrate from libhandy-0.1 to libhandy-1

libhandy-0.1 is a really old version and most apps have migrated to
libhandy-1 by now. In Alpine in particular, Thumbdrives is the only
package which still depends on libhandy-0.1. As such, migrate it to the
newer version.
---
 meson.build   | 2 +-
 src/window.py | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/meson.build b/meson.build
index a44ec0c..39a5d35 100644
--- a/meson.build
+++ b/meson.build
@@ -5,7 +5,7 @@ project('thumbdrives',
                    ],
 )
 
-dependency('libhandy-0.0', version: '>=0.0.10')
+dependency('libhandy-1')
 
 i18n = import('i18n')
 
diff --git a/src/window.py b/src/window.py
index 6223ccc..81d4097 100644
--- a/src/window.py
+++ b/src/window.py
@@ -4,13 +4,13 @@ from pathlib import Path
 
 import gi
 gi.require_version('Gtk', '3.0')
-gi.require_version('Handy', '0.0')
+gi.require_version('Handy', '1')
 from gi.repository import Gtk, Handy
 import xdg.BaseDirectory
 
 import thumbdrives.vdisk as vdisk
 
-Handy.Column()
+Handy.Clamp()
 
 @Gtk.Template(resource_path='/nl/brixit/Thumbdrives/window.ui')
 class ThumbdrivesWindow(Gtk.ApplicationWindow):
-- 
2.44.0

